tools/nsp_scripts/test_model_script.py

```diff
@@ -15,31 +22,331 @@
 from droidlet.perception.semantic_parsing.nsp_transformer_model.decoder_with_loss import *
 from droidlet.perception.semantic_parsing.nsp_transformer_model.encoder_decoder import *
 from droidlet.perception.semantic_parsing.nsp_transformer_model.caip_dataset import *
+from droidlet.perception.semantic_parsing.load_and_check_datasets import get_ground_truth
+from droidlet.perception.semantic_parsing.nsp_transformer_model.utils_caip import caip_collate
+from droidlet.perception.semantic_parsing.utils.nsp_logger import NSPLogger
+
+GT_QUERY_ACTIONS = get_ground_truth(
+    False, 
+    "droidlet/artifacts/datasets/ground_truth/"
```

Comment:
note that you are passing in an absolute path here, you should either add relative path with respect to the folder this script will be run from or mention this in a README like file for this script